❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao sincronizar funcionários: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro na sincronização: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao sincronizar funcionários: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro na sincronização: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar status das pesquisas: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\connection-pool.js:85:17
    at Connection.onConnect (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:849:9)
    at Object.onceWrapper (node:events:633:26)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:16
    at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
  code: 'EINSTLOOKUP',
  originalError: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
      at C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:1006:32
      at process.processTicksAndRejections (node:internal/process/task_queues:85:11) {
    code: 'EINSTLOOKUP',
    [cause]: Error: getaddrinfo ENOTFOUND lmc00sv006
        at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:122:26) {
      errno: -3008,
      code: 'ENOTFOUND',
      syscall: 'getaddrinfo',
      hostname: 'lmc00sv006'
    }
  }
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao atualizar usuário 000001: RequestError: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
    at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:384:15)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
    at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
    at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22) {
  code: 'EREQUEST',
  originalError: Error: Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". 
      at handleError (C:\inetpub\LumiGente\LumiGente-main\node_modules\mssql\lib\tedious\request.js:382:19)
      at Connection.emit (node:events:518:28)
      at Connection.emit (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\connection.js:970:18)
      at RequestTokenHandler.onErrorMessage (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\handler.js:284:21)
      at Readable.<anonymous> (C:\inetpub\LumiGente\LumiGente-main\node_modules\tedious\lib\token\token-stream-parser.js:19:33)
      at Readable.emit (node:events:518:28)
      at addChunk (node:internal/streams/readable:561:12)
      at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
      at Readable.push (node:internal/streams/readable:393:5)
      at nextAsync (node:internal/streams/from:194:22) {
    info: ErrorMessageToken {
      name: 'ERROR',
      handlerName: 'onErrorMessage',
      number: 7346,
      state: 2,
      class: 16,
      message: 'Cannot get the data of the row from the OLE DB provider "OraOLEDB.Oracle" for linked server "ORACLE_PROD_SJP". ',
      serverName: 'LMC00SV006\\DWLUMICENTER',
      procName: 'sp_SyncUserHierarchy',
      lineNumber: 17
    }
  },
  number: 7346,
  lineNumber: 17,
  state: 2,
  class: 16,
  serverName: 'LMC00SV006\\DWLUMICENTER',
  procName: 'sp_SyncUserHierarchy',
  precedingErrors: []
}
❌ Erro ao verificar status de avaliações: ConnectionError: getaddrinfo ENOTFOUND lmc00sv006
